tests/unit/workspace_access/test_generic.py

```diff
@@ -925,4 +925,14 @@ def test_serving_endpoints_not_enabled_raises_warning(caplog):
     sup = GenericPermissionsSupport(ws=ws, listings=[Listing(ws.serving_endpoints.list, "id", "serving-endpoints")])
     with caplog.at_level('WARNING'):
         list(sup.get_crawler_tasks())
-    assert "Listing serving-endpoints failed: Model serving is not enabled for your shard" in caplog.text
+    assert "Listing serving-endpoints failed" in caplog.text
+
+
+def test_internal_error_in_serving_endpoints_raises_warning(caplog):
```

Comment:
```suggestion
def test_internal_error_in_serving_endpoints_raises_warning(caplog) -> None:
```

Otherwise the test is not covered by mypy

Reply:
Shouldn't we add the same output format to the other unit tests with caplog?

Reply:
It is tech debt, should be added to any test missing type hints. You do not have to tackle it in this PR. If you do, please only update the tests that are close to your changes